lib/mods_display/fields/date_other.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module ModsDisplay
+  class DateOther < Field
+    def fields
+      date_fields(:dateOther)
+    end
+
+    private
+
+    # overriding the original method in Field in order to include the type attribute value in parens
+    def select_best_date(dates)
+      # ensure dates are unique with respect to their base values
+      dates = dates.group_by(&:base_value).map do |_value, group|
+        group.first if group.one?
+
+        # if one of the duplicates wasn't encoded, use that one. see:
+        # https://consul.stanford.edu/display/chimera/MODS+display+rules#MODSdisplayrules-3b.%3CoriginInfo%3E
+        if group.reject(&:encoding).any?
```

Comment:
This entire `if` could be consolidated into `group.reject(&:encoding).first || group.last`.

Reply:
Good refactor, thanks.   I've been a bit too hesitant to do these sorts of refactors as I've encountered them.